packages/myst-cli/src/build/cff.ts

```diff
@@ -40,19 +41,27 @@ export async function runCffExport(
   let abstract: string | undefined;
   if (projectPath && selectors.selectLocalConfigFile(state, projectPath) === sourceFile) {
     frontmatter = selectors.selectLocalProjectConfig(state, projectPath);
+    // TODO: This uses project-config-level parts which are not yet supported in the same way
+    // as page- and site-level parts. Once those are supported, this needs updating.
     const rawAbstract = frontmatter?.parts?.abstract?.join('\n\n');
```

Comment:
This is one place where we already use `project` parts - it has to be handled in the old way still

Reply:
Project part processing has been updated, along with the logic here.